solidity/dashboard/src/actions/index.js

```diff
@@ -11,3 +11,35 @@ export const RESET_COPY_STAKE_FLOW = "copy-stake/reset_flow"
 export const SET_STRATEGY = "copy-stake/set_strategy"
 export const SET_DELEGATION = "copy-stake/set_delegation"
 export const COPY_STAKE_REQUEST = "copy-stake/copy_stake_request"
+
+// OPERATOR
+export const FETCH_OPERATOR_DELEGATIONS_RERQUEST =
+  "operator/fetch_delegations_request"
+export const FETCH_OPERATOR_DELEGATIONS_START =
+  "operator/fetch_delegations_start"
+export const FETCH_OPERATOR_DELEGATIONS_SUCCESS =
+  "operator/fetch_delegations_success"
+export const FETCH_OPERATOR_DELEGATIONS_FAILURE =
+  "operator/fetch_delegations_failure"
+export const FETCH_OPERATOR_SLASHED_TOKENS_RERQUEST =
+  "operator/fetch_slashed_tokens_request"
+export const FETCH_OPERATOR_SLASHED_TOKENS_START =
+  "operator/fetch_slashed_tokens_start"
+export const FETCH_OPERATOR_SLASHED_TOKENS_SUCCESS =
+  "operator/fetch_slashed_tokens_success"
+export const FETCH_OPERATOR_SLASHED_TOKENS_FAILURE =
+  "operator/fetch_slashed_tokens_failure"
+export const OPERATOR_DELEGATION_UNDELEGATED = "operator/delegation_undelegated"
+export const OPERATR_DELEGATION_CANCELED = "operator/delegation_canceled"
```

Comment:
Just a note: I think it would be more readable if we keep all action names in the same tense 🤔 As far as i see we usually use past simple so maybe it would be better to change it to _REQUESTED, _STARTED, _SUCCESS?

Although if I can be honest I would prefer to keep it all in present simple 😅 

EDIT: Actually I see that we used present simple in other places also for START, SUCCESS and FAILURE, but still maybe we could refactor and use only one tense in the near future?